lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -387,13 +467,6 @@ export abstract class CacheManager implements ICacheManager {
                 return;
             }
 
-            if (
-                !!accountFilter.localAccountId &&
```

Comment:
Just confirming these are being removed because they've been moved somehwere else?

Reply:
Yes, these are now part of the ID token claims fitlering that's done when the tenant profile is build (once the base account info has been matched to the filter)